content/en/agent/proxy.md

```diff
@@ -90,6 +94,10 @@ The Agent uses the following values in order of precedence:
 {{% /tab %}}
 {{% tab "Agent v5" %}}
 
+<div class="alert alert-warning">
+The <code>&ltHOST&gt;:&ltPORT&gt;</code> used to proxy metrics can NOT be used to proxy logs.
```

Comment:
Should we explain why? The reason being that logs are forwarded through TCP.

Reply:
Same